migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -127,7 +127,9 @@ For more information, see xref:../storage/understanding-persistent-storage.adoc#
 
 {product-title} 4 is migrating in-tree volume plug-ins to their Container Storage Interface (CSI) counterparts. In {product-title} {product-version}, CSI drivers are the new default for the following in-tree volume types:
 
+* Amazon Web Services (AWS) Elastic Block Storage (EBS)
 * Azure Disk
+* Google Compute Engine Persistent Disk (GCP PD)
```

Comment:
@jsafrane Is "Google Compute Engine" here correct? (pulled this based on the docs at https://docs.openshift.com/container-platform/4.12/storage/container_storage_interface/persistent-storage-csi-migration.html#persistent-storage-csi-migration-automatic-ga_persistent-storage-csi-migration

And if so, shouldn't the abbreviation be "GCE" instead of "GCP" here?

Reply:
Thanks @jsafrane, I've updated to `Google Cloud Platform Persistent Disk (GCP PD)`.

FYI @lpettyjo in case you want to update https://docs.openshift.com/container-platform/4.12/storage/container_storage_interface/persistent-storage-csi-migration.html#persistent-storage-csi-migration-automatic-ga_persistent-storage-csi-migration (and any other places in the repo) to use `Google Cloud Platform Persistent Disk (GCP PD)` too.